packages/zoe/src/internal-types.js

```diff
@@ -14,6 +14,7 @@
  * @property {Notifier<Allocation>} notifier
  * @property {Allocation} initialAllocation
  * @property {SeatHandle} seatHandle
+ * @property {Object=} offerArgs
```

Comment:
I'm not sure if there's a proper way in TypeScript to distinguish between "this property may not exist" vs "this property's value may be undefined". In my ignorance of TypeScript, when I see {Object=} I leap to the first. But here you mean the second. How about
```suggestion
 * @property {Object | undefined} offerArgs
```

Reply:
I think `Object=` is what is recommended for saying that a property may be undefined.

![Screen Shot 2021-08-03 at 4 59 47 PM](https://user-images.githubusercontent.com/2441069/128101082-4ff287ec-d014-480a-86c7-61f67730ad6e.png)

Reply:
Hmm, but specifying it explicitly as you do, means that the `?` is dropped:

![Screen Shot 2021-08-03 at 5 01 29 PM](https://user-images.githubusercontent.com/2441069/128101408-e6fab635-59d2-4b01-b1a5-1b43c7ea72ef.png)




I wonder whether this has any meaningful difference, though, as doesn't nearly everything treat a property that doesn't exist and a property that exists but is undefined, the same?

Reply:
I'm going to leave this as is for conformity with the rest of Zoe but we can continue this conversation.